Source/DafnyLanguageServer.Test/Completion/AtCompletionTest.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.Dafny.LanguageServer.IntegrationTest.Extensions;
+using OmniSharp.Extensions.LanguageServer.Protocol.Document;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Dafny.LanguageServer.IntegrationTest.Util;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Completion {
```

Comment:
You should have at least one test case where `@` is the very first character of the file (i.e. not even any comment first), since that's a logic edge case.